source/user-manual/manager/automatic-reports.rst

```diff
@@ -7,7 +7,7 @@
 
 Generating automatic reports
 -----------------------------
-Daily reports are summaries of the alerts that were triggered each day. You can configure your own customized report by using the ``report`` option in the ``ossec.conf`` file. More information on this can be found at :ref:`Report <reference_ossec_reports>`. For information on how to configure email alerts , see the :ref:`Configuring email alerts <manual_email_report>` and :ref:`SMTP server with authentication <smtp_authentication>` sections for reference.
+Daily reports are summaries of the alerts that were triggered each day. You can configure your own customized report by using the ``report`` option in the ``ossec.conf`` file. More information on this can be found at :ref:`Report <reference_ossec_reports>`. For information on confining email alerts, see the :ref:`Configuring email alerts <manual_email_report>` and :ref:`SMTP server with authentication <smtp_authentication>` sections for reference.
```

Comment:
```suggestion
Daily reports are summaries of the alerts that were triggered each day. You can configure your own customized report by using the ``report`` option in the ``ossec.conf`` file. More information on this can be found at :ref:`Report <reference_ossec_reports>`. For information on configuring email alerts, see the :ref:`Configuring email alerts <manual_email_report>` and :ref:`SMTP server with authentication <smtp_authentication>` sections for reference.
```